### DISCUSSION TITLE
Is primeicons alive?

### DISCUSSION THREAD

**Discussion by inad9300** [primefaces/community#1393] [PrimeIcons] [answered]
In multiple occasions I have tried to ask about the state of this library, unsuccessfully. This is one more attempt to see if I have more luck than in the past. I'm not asking for any action to be taken, just for information to be provided.

As it stands today, the icon library is very limited, as made evident by the hundreds of user requests. Yet, since it already comes with PrimeReact et al., ideally one would like to use it, before introducing an extra dependency. So, I would like to know: are you planning on ever extending it again, or is it hopeless for us to keep expecting this?

**Comment by onursenture** [comment, marked as answer]
Hi, PrimeIcons is a project we started in 2018 and have been updating regularly every year. There are two main reasons why we don't want to incorporate all icon requests into the library:

- Most icon additions may not be meaningful for all other users.
- Each icon addition increases the bundle size, which UI library users don't perceive as positive.

As of last year, we implemented templating features in all our UI libraries to facilitate the use of custom icon libraries. This way, you can use any library you wish, independent of PrimeIcons. You can refer to this link: https://primereact.org/customicons/

For the future, yes, PrimeIcons is not one of our main projects, and it won't receive updates as frequently as our UI libraries. However, we will still look at icon requests and make updates.

Thank you for being so interested.
Best regards,
